File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result={0,1};
        for(int i=0;i<nums.length;i++){
            for(int j=0;j<nums.length;j++){
                int sum=nums[i]+nums[j];
                if(sum==target && i!=j){
                    result[0]=i;
                    result[1]=j;
                    return result;
                }
            }
        }
        
        return result;
        
    }
}

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen ={}
        for i, value in enumerate(nums):
            if value in seen:         #check if value is required for previous element
                return[seen[value],i] #return previos elemet and current index
            else: 
                seen[target-value]=i #store required pair for ith element